pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/PropertiesToMapTreeUtil.java

```diff
@@ -33,7 +33,7 @@ public static Map<String, Object> propertiesToMap(Map<String, Object> properties
             Map<String, Object> valueMap = createTree(keyList, map);
             Object value = properties.get(key);
             if (value instanceof String) {
-                value = StringEscapeUtils.unescapeHtml((String)value);
+                value = StringEscapeUtils.unescapeHtml4((String)value);
```

Comment:
I _think_ this is the best choice, since `unescapeHtml` is no longer available in the updated `StringEscapeUtils`.